### ISSUE TITLE
Support for UNSIGNED values in $createTable?

### ISSUE THREAD

**Issue by lukesrw** [planetarydev/json-sql-builder2#28]
Is there any support, or planned support for unsigned values currently?

I've searched through the code and I'm not finding any implementation.

**Comment by planetarydev** [comment]
No, currently there are no plans. But you can do it yourself very easily if you like to add it like the notnull implementation. 
Everything happens in the [$colum helper](https://github.com/planetarydev/json-sql-builder2/tree/master/sql/helpers/ddl/column).

**Comment by planetarydev** [comment]
@lukesrw some news on that?
